hyperspy/signal.py

```diff
@@ -2656,7 +2656,13 @@ def get_1D_sum_explorer_wrapper(*args, **kwargs):
             navigator = self
             # Sum over all but the first navigation axis.
             am = navigator.axes_manager
-            navigator = navigator.sum(am.signal_axes + am.navigation_axes[1:])
+            with warnings.catch_warnings():
+                warnings.filterwarnings("ignore", category=UserWarning,
+                                        module='hyperspy'
+                                        )
+                navigator = navigator.sum(
```

Comment:
It would make sense to use integrate1D here, which correctly decides on whether a `sum` can be made (binned signal) or an integration is needed (esp. non-uniform, unbinned signal)  -- in fact, it is on the todo-list in #2398. Then we could avoid catching the warning here.

Reply:
Is there a reason to use `integrate1D` in favour of `sum`?

Using `integrate_simpson` will not work for `len(am.signal_axes + am.navigation_axes[1:] > 1` where `am` is the `axes_manager`.

The following example will not work:
```python
import hyperspy.api as hs
import numpy as np

s = hs.signals.Signal2D(np.arange(5*10*15).reshape(5, 10, 15))
s.plot(navigator='spectrum')
```

Reply:
The idea is to preserve the physical meaning of the signal. But I agree that this is out of reach for a simple PR like this one.

The objective of this pull request is just to fix some faulty behaviour in `BaseDataAxis.value2index()` (I updated the description to make this clear) but I am fine with attacking the `Integrate1D` question in another PR

Reply:
Yes, that was the idea - but probably you're right to leave this one for later.